tdrs-frontend/src/components/SiteMap/SiteMap.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useRef, useState } from 'react'
+
+const SiteMap = () => (
```

Comment:
Maybe, just add a note here to remind people to update the tests when a new link is added